features/before_after_all_hook_timeouts.feature

```diff
@@ -29,7 +29,8 @@ Feature: before / after all hook timeouts
     Then it fails
     And the error output contains the text snippets:
       | a handler errored, process exiting        |
-      | function timed out after 500 milliseconds |
+      | function timed out                        |
+      | 500 milliseconds                          |
```

Comment:
Can we update this to: `function timed out, ensure the callback is executed within 500 milliseconds`?